.appveyor.yml

```diff
@@ -56,6 +56,7 @@ build: false
 test_script:
   - cargo build --verbose
   - cargo test --verbose
```

Comment:
That's fair. I should say that I did it this way intentionally, to make sure that the crate still builds and tests correctly *without* the serde feature enabled (making sure I set up my `cfg` attributes correctly etc). But with enough of these kinds of features it would lead to ridiculous test times, having to run through the test suite several times... so yeah, I can remove it :)